### ISSUE TITLE
Interpreting telomere length estimates from Telogator2

### ISSUE THREAD

**Issue by niyati1211** [zstephens/telogator2#7]
Hello,

I have a question about interpreting results from Telogator2. Should I use the value from the TL_p75 column to report the telomere length for my samples? I am asking because it seems that the y-axis (ATL) of the violin plots generated by Telogator2 is plotting values equivalent to TL + TVR_len. According to my understanding of the paper, ATL should correspond to just the telomere length. Could you please clarify which value should be reported?

Thank you!!

**Comment by zstephens** [comment]
You're right that the violin plots use the TL + TVRlen values, in part because I don't think there's consensus yet on whether TVR regions contribute to effective TL or not. E.g. [Tham et al.](https://www.nature.com/articles/s41467-023-35823-7) choose to define the telomere as the first instance where ~2 tandem copies of the canonical repeat are found, which will include nearly the entire TVR region in most cases.

Ideally, the distribution of ATLs at each allele (i.e. the TLs from all of its supporting reads) represents the distribution of TLs across different cells in the cell population from which the DNA was derived. And how that distribution gets summarized to a single value might depend on your use case. We chose to make the 75th percentile ATL (TL_p75) the default output as it was the most consistent metric when doing repeated runs of the same sample, and it's the value I'd recommend using when comparing across samples.

**Comment by niyati1211** [comment]
Thank you so much! This is really helpful.
